composer.json

```diff
@@ -15,7 +15,8 @@
         "doctrine/data-fixtures": "1.*@alpha",
         "doctrine/doctrine-fixtures-bundle": "2.*@alpha",
         "doctrine/orm": "2.*",
-        "doctrine/doctrine-bundle": "*"
+        "doctrine/doctrine-bundle": "*",
+        "symfony/property-access": "~2.3"
```

Comment:
no, it's allow all version >= 2.3, i have chosen the 2.3 because 2.2 is deprecated version.